LiskHQ/lisk-desktop
# Only increase the nonce one time in edit multi sig flow

### What was the problem?

This PR resolves #5440

### How was it solved?

Dont increase nonce of transactionAuthor if TxParamMembers is empty

### How was it tested?

1. Create multisig account for Account_1
2. Try to send a transaction from Account_1
3. Expected: Nonce reset should not be displayed in transaction summary
